### ISSUE TITLE
Tests/Python 3: update unittest syntax

### ISSUE THREAD

**Issue by josephsl** [nvaccess/nvda#9720]
Hi,

In order to test this, one must be using threshold_py3_staging branch.

### Steps to reproduce:
1. Checkout latest threshold_py3_staging work.
2. Perform scons tests.

### Actual behavior:
Errors related to syntax in unittest test cases are thrown.

### Expected behavior:
No errors

#### NVDA installed/portable/running from source:
Source

#### NVDA version:
Python 3 source

#### Windows version:
Windows 10 Version 1903 (build 18362)

#### Name and version of other software in use when reproducing the issue:
None

#### Other information about your system:
None

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes - no errors in Python 2 version.

### Additional notes:
Scons tests revealed numerous source code errors, including removal of itertools.izip and others. I'll take care of izip issue as part of another pull request.

Thanks.

**Comment by josephsl** [comment]
Hi,

The specific issues are use of unicode() function, no relative import, assertItemsEqual -> assertCountEqual and others.

Thanks.

**Comment by josephsl** [comment]
blocked by #9721

**Comment by leonardder** [comment]
Fixed in #9726
